scripts/init_opa.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+apk update
+apk add curl
+
+curl -s -i -X 'GET' \
```

Comment:
can't we iterate over the api and extract the list of tenants from which policies needs to be extracted?